lib/deploy/activate.rb

```diff
@@ -42,7 +42,7 @@ def clone_idp_config
       )
       checkout_dir = File.join(root, idp_config_checkout_name)
 
-      cmd = ['git', 'clone', private_git_repo_url, checkout_dir]
+      cmd = ['git', 'clone', '--depth', '1', '--branch', 'main', private_git_repo_url, checkout_dir]
```

Comment:
Do we need `--branch` here?

Reply:
Nope, it’s not strictly needed since it will default to main. I went with explicitly naming it to act as a placeholder should it ever be configurable.